### PR TITLE
Add Ultra Beasts and more Zygarde aliases

### PR DESCRIPTION
**Pull request by Ridaz**

### CODE REVIEW

File: data/aliases.js [hunk 1]
```diff
@@ -428,6 +430,27 @@ exports.BattleAliases = {
...
+	"absorption": "Buzzwole",
+	"beauty": "Pheromosa",
+	"blade": "Kartana",
+	"blaster": "Celesteela",
+	"expansion": "Buzzwole",
+	"gluttony": "Guzzlord",
+	"lightning": "Xurkitree",
+	"symbiont": "Nihilego",
+	"ub01": "Nihilego",
+	"ub01symbiont": "Nihilego",
```
Comment (TheImmortal) [inline data/aliases.js]: I don't think anyone is going to use this or the other full UB names...

File: data/aliases.js [hunk 2]
```diff
@@ -428,6 +432,25 @@ exports.BattleAliases = {
 	"ohmagod": "Plasmanta",
 	"zong": "Bronzong",
 
+	// ultra beasts
+	"absorption": "Buzzwole",
+	"beauty": "Pheromosa",
+	"blade": "Kartana",
+	"blaster": "Celesteela",
+	"expansion": "Buzzwole",
+	"gluttony": "Guzzlord",
```
Comment (Marty-D) [inline data/aliases.js]: The English name for it is Glutton, which is lucky for us since Gluttony is already a thing.

**Comment by Ridaz** [comment]
@Marty-D you can merge it now

File: data/aliases.js [hunk 3]
```diff
@@ -428,6 +433,25 @@ exports.BattleAliases = {
...
+	"absorption": "Buzzwole",
+	"beauty": "Pheromosa",
+	"blade": "Kartana",
+	"blaster": "Celesteela",
+	"expansion": "Buzzwole",
+	"glutton": "Guzzlord",
+	"lightning": "Xurkitree",
+	"symbiont": "Nihilego",
+	"ub01": "Nihilego",
+	"ub02absorption": "Buzzwole",
```
Comment (TheImmortal) [inline data/aliases.js]: you didn't remove this, or the other ones...

Comment (Ridaz) [inline data/aliases.js]: Remember that there are two UB-02 and two UB-04. I guess I can remove UB-03 Lightning

File: data/aliases.js [hunk 4]
```diff
@@ -428,6 +433,16 @@ exports.BattleAliases = {
...
 
+	// ultra beasts
+	"ub01": "Nihilego",
+	"ub02absorption": "Buzzwole",
+	"ub02beauty": "Pheromosa",
+	"ub03": "Xurkitree",
+	"ub04blade": "Kartana",
+	"ub04blaster": "Celesteela",
+	"ub05": "Guzzlord",
+	"ub05gluttony": "Guzzlord",
```
Comment (Marty-D) [inline data/aliases.js]: You only changed one of them.

File: data/aliases.js [hunk 5]
```diff
@@ -428,6 +433,16 @@ exports.BattleAliases = {
 	"ohmagod": "Plasmanta",
 	"zong": "Bronzong",
 
+	// ultra beasts
+	"ub01": "Nihilego",
+	"ub02absorption": "Buzzwole",
```
Comment (TheImmortal) [inline data/aliases.js]: this is still here...

File: data/aliases.js [hunk 6]
```diff
@@ -428,6 +433,16 @@ exports.BattleAliases = {
...
+	"ub02beauty": "Pheromosa",
```
Comment (TheImmortal) [inline data/aliases.js]: and ub02b.

File: data/aliases.js [hunk 7]
```diff
@@ -428,6 +433,16 @@ exports.BattleAliases = {
...
+	"ub03": "Xurkitree",
+	"ub04blade": "Kartana",
```
Comment (TheImmortal) [inline data/aliases.js]: I don't know what to do here...

File: data/aliases.js [hunk 8]
```diff
@@ -428,6 +433,16 @@ exports.BattleAliases = {
...
+	"ub04blaster": "Celesteela",
+	"ub05": "Guzzlord",
+	"ub05glutton": "Guzzlord",
```
Comment (TheImmortal) [inline data/aliases.js]: this can be removed

File: data/aliases.js [hunk 9]
```diff
@@ -201,8 +201,13 @@ exports.BattleAliases = {
 	"floetteeternalflower": "Floette-Eternal",
 	"ashgreninja": "Greninja-Ash",
 	"zydog": "Zygarde-10%",
+	"zydoge": "Zygarde-10%",
```
Comment (TheImmortal) [inline data/aliases.js]: we adding memes now?

File: data/aliases.js [hunk 10]
```diff
@@ -201,8 +201,13 @@ exports.BattleAliases = {
...
+	"zygardedog": "Zygarde-10%",
 	"zygarde50": "Zygarde",
+	"zyc": "Zygarde-Complete",
 	"zygarde100": "Zygarde-Complete",
+	"zygardec": "Zygarde-Complete",
+	"zygod": "Zygarde-Complete",
```
Comment (TheImmortal) [inline data/aliases.js]: do people even use this? I could see something like "zygardefull" being used

Comment (Ridaz) [inline data/aliases.js]: Apparently in OM it's used quite a bit

[21:55:16] partys over ?_: ive heard zygod for complete
[21:55:31] @Ridaz: where
[21:55:37] partys over ?_: friends that play bh
[21:55:38] Macch Aeger: it's pretty common in the om room
[21:56:31] +ih8ih8sn0w: zygod is pretty common in oms, yeah (and ive probably tried to do !dt zygod on a couple occasions)

File: data/aliases.js (hunk 9 above)
Comment (Ridaz) [inline data/aliases.js]: Apparently it's a thing, even though I don't appreciate shit ones

[21:53:21] +Kris: zydoge is another alias
[21:53:47] @Ridaz: really
[21:53:52] +Kris: yes really
[21:55:05] @Ridaz: tfw shit meme will never end

**Pull request merged**